### DISCUSSION TITLE
Limit on Write Collaborators | Github Organization (Free)

### DISCUSSION THREAD

**Discussion by AntoineRZGH** [community/community#140992] [New to GitHub] [answered]
Hi,

I want to create a Free Github Organization account, where I would have several private repositories.

Are there any limits on the number of (free) members that can join the private repositories with write permissions? (I am mostly concerned about organization members; not external collaborators). 
I am being told that there is , but I can't seem to find anything in Github's documentation that says so...

**Comment by INAHIDC** [comment]
i dont think theres any limit here, you can create unlimited private repositories with unlimited collaborators!!

**Reply by AntoineRZGH** [reply]
Thank you very much for your answer!

**Comment by ValentinoSpirito** [comment, marked as answer]
Hey Pointing9212,

Great question! For a free GitHub Organization account, you can have unlimited private repositories, and there's no limit on the number of members within the organization. However, the distinction comes with permissions—free organization accounts can add as many members as they want, but those members all receive the same default permissions.

If you're adding members to your organization, they will all get read or write permissions for private repositories depending on how you set up access. GitHub doesn’t explicitly limit the number of these organization members, but you might run into practical management issues if the group gets too large without paid features for more granular control.

Hope that clears things up! Let me know if you need any more details.

Best,
Valentino

**Reply by AntoineRZGH** [reply]
Thank you very much for your detailed answer!

**Reply by ghostinhershell** [reply]
Hi @Pointing9212 👋🏾,It looks like @ValentinoSpirito's response was the answer to your question here. Can you please mark their response as the answer? When you mark a question as an answer, GitHub will highlight the comment and replies to the comment to help visitors quickly find the answer.
![image](https://github.com/user-attachments/assets/b89e7161-c2d1-47e2-a058-242df633e6b2)
